conans/test/unittests/client/tools/system_pm_test.py

```diff
@@ -150,6 +157,18 @@ def system_package_tool_test(self):
             spt.update()
             self.assertEqual(runner.command_called, "sudo -A apt-get update")
 
+            # Fake a dnf exe and check if for fedora it used it
```

Comment:
I'm ok leaving the fake executable, but we can leave a FIXME so anyone motivated can improve it mocking the system call.

```suggestion
            # Fake a dnf executable and check Fedora uses it.  # FIXME: Mock which("dnf") function
```

Reply:
It would say `Mock which("dnf") function` if you dare. I'm pleased if you explain it to me how to do it.